docs/application/dotnet/get-started/first-app.md

```diff
@@ -50,20 +54,20 @@ To create a new Tizen .NET project, you can use following tools:
                     <ol>
                         <li>Launch the Visual Studio tool.</li>
                         <li>In the Visual Studio menu, select <b>File</b> > <b>New</b> > <b>Project</b>.</li>
-                        <img alt=" " src="media/vs2019_project_create_1.png" />
+                        <img alt="Create new project" src="media/vs2019_project_create_1.png" />
                         <p></p>
-                        <li>In <b>New project</b> menu, select <b>Tizen</b> > <b>Blank App (Tizen.NUI)</b>. Then click <b>Next</b>.</li>
-                        <img alt=" " src="media/vs2019_project_create_2.png" />
+                        <li>In <b>New project</b> menu, select <b>#C</b> and <b>Tizen</b> from the dropdown menu, and choose <b>Blank App (Tizen.NUI)</b>. Then click <b>Next</b>.</li>
+                        <img alt="New project menu" src="media/vs2019_project_create_2.png" />
                         <p></p>
                         <p>Configure the project properties and click <b>Create</b>. You can enter the <b>Project name</b>, <b>Location</b>, and <b>Solution name</b>.</p>
-                        <img alt=" " src="media/vs2019_project_create_3.png" />
+                        <img alt="Configure your project" src="media/vs2019_project_create_3.png" />
                         <p></p>
                         <p>The <b>Tizen Project Wizard</b> pop-up window appears.</p>
                         <li>Select <b>Platform Version</b> and click <b>OK</b></li>
-                        <img alt=" " src="media/vs2019_project_create_4.png" />
+                        <img alt="Tizen Project Wizard" src="media/vs2019_project_create_4.png" />
```

Comment:
@mijin85cho Sure, I'll install VS tools and update the screenshots!

Reply:
@mijin85cho When installing Visual Studio and the Tizen extension, I found a few problems. For Tizen 6.5, Visual Studio 2022 is required, so the information needs to be updated for both `first-app.md` and `vstools/install.md` (Visual Studio 2022 in place of Visual Studio 2019).

I also faced problems while installing the Tizen extension, so couldn't properly take all the screenshots. For now I am making changes according to the other comments you have posted here.

What do you think should be done regarding the Visual Studio issues in the article?

Reply:
Hello, @safir-srbd 
Thanks for the trying. What was the problem you faced while installing the Tizen extension?
I think @dkson95 or @sangwook96-lee can take a look if you share us what the problem was

Reply:
@mijin85cho I installed Visual Studio 2022 Community Edition (without all the packages, as it would've been a large download), and installed the Tizen Extension. But for some reason, the extension didn't load properly, and I couldn't complete the creation of projects in the app. Is it because I didn't download the extra packages?

Reply:
@safir-srbd
Hi,
Could you follow the steps " https://docs.tizen.org/application/vstools/install/"?

Reply:
@sangwook96 I followed the steps and installed accordingly, but did not install the recommended packages, as they were very large in size (12GB).

Reply:
@safir-srbd 
Hi, yes, it's a large size

Reply:
@sangwook96 I will try again and download the recommended packages to see if it works with that.....

Reply:
@mijin85cho @sangwook96 I have managed to get the extension to work, which allowed me to take the screenshots required.

However, these screenshots were taken from Visual Studio 2022, and currently in the `first-app.md` file, the screenshots are placed in a section titled Visual Studio 2019. Should I update the heading text to Visual Studio 2022, as it is the minimum required version required to work with Tizen 6.5, or keep it as it is?

Reply:
> @mijin85cho @sangwook96 I have managed to get the extension to work, which allowed me to take the screenshots required.
> 
> However, these screenshots were taken from Visual Studio 2022, and currently in the `first-app.md` file, the screenshots are placed in a section titled Visual Studio 2019. Should I update the heading text to Visual Studio 2022, as it is the minimum required version required to work with Tizen 6.5, or keep it as it is?

Good point. As the minimum required version will be changed to 2022 (for 6.5 and higher) I think changing to 2022 is better. Also the screenshots are taken in 2022, too.

@sangwook96 Would you please let me know your thought?

Reply:
@mijin85cho @sangwook96 hope you are doing well.

Any updates on what steps should I take regarding this issue?

Reply:
@safir-srbd,
Hi
Sorry for being late updating my answer.
I think that it could keep it as it is because the developers may understand what we say with even the old images.
Thank you very much for giving the good suggestion but please keep it as it in this case.
Thanks

Reply:
Hello, @sangwook96 
I worry that it seems there is some miscommunication between the question and the answer.

**To my understanding,**
with your help @safir-srbd captured the screenshot in new 2022 version.
However, the table title in this document is written as "Visual Studio 2019" for Visual Studio,
even we will use the Visual Studio 2022 screenshot

For example you can see the "Visual Studio 2019" mark in the table in 
https://docs1.stg.tizen.org/staging/1627/application/dotnet/get-started/first-app/#create-a-project

So, @safir-srbd 's question is "Can we change the "Visual Studio 2019" table heading to "Visual Studio 2022""? 

In my opinion, as we are going to use the screenshot from "Visual Studio 2022",
the table title can be changed to "Visual Studio 2022".
(Or maybe we can just use "Visual Studio")

If I understood something wrong, please let me know.

Reply:
@mijin85cho I think using "Visual Studio" would be best here, as it would make the heading correct no matter what version of screenshot is present.